qbnb__test/test_models.py

```diff
@@ -0,0 +1,105 @@
+import sys
+sys.path.insert(1, "C:\\Users\\adlai\\Desktop\\CISC327\\qbnb\\qbnb\\")
+from qbnb.models import user_register
+from models import *
+import requests
+from mongoengine import *
+
+def test_r1_1_user_register():
+    '''
+    Testing R1-1: Email cannot be empty. Password Cannot be empty.
+    '''
+    user0 = user_register("user1@gmail.com", "Password1!", "user1")
+    user1 = user_register("", "", "user1")
+    
+    assert user0 is True
+    assert user1 is False
+
+
+def test_r1_2_user_register():
+    '''
+    Testing R1-2: A user is uniquely identified by his/her 
+    user id - automatically generated.
+    '''
+    user = user_register("user1@gmail.com", "Password1", "user1")
+    assert user._id is not None
```

Comment:
This does not check if the user id is unique, because User.id is assigned manually and is not the id of the document in MongoDB (which is unique).